packages/xo-server-usage-report/src/index.js

```diff
@@ -426,7 +436,7 @@ function computeGlobalVmsStats({ haltedVms, vmsStats, xo }) {
       'netTransmission',
     ]),
     {
-      number: allVms.length,
+      number: vmsStats.length + haltedVms.length,
```

Comment:
Why this change?

Reply:
Because `allVms` not contains replicated VMs anymore. That's why we shouldn't be base it on it to calculate the number  of existent VMs.